src/style.css

```diff
@@ -1,3 +1,84 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
 body {
+  --color: #00acdc;
+
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+header {
+  width: 100%;
+  min-width: 1024px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
   margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: inline-block;
+  height: 60px;
+  text-decoration: none;
+  color: black;
+  position: relative;
+}
+
+.nav__item {
+  display: flex;
+  height: 15px;
+  text-transform: uppercase;
+  line-height: 15px;
+  font-weight: 500;
+  margin-right: 20px;
+}
+
+.menu {
+  height: 60px;
+  padding-right: 40px;
+  padding-left: 150px;
+  margin-left: 50px;
+  margin-top: 20px;
+}
+
+img {
+  padding-left: 50px;
+  margin-top: 10px;
+}
```

Comment:
Avoid using tag names for styling. It's better to use a class for the 'img' tag to apply styles. This ensures that if you have other images on the page that require different styling, you can easily manage that by using different classes.